share/revng/test/tests/abi/build_abi_test_binaries_with_gcc.sh

```diff
@@ -31,12 +31,20 @@ mkdir -p "${OUTPUT_DIRECTORY}"
   -o "${OUTPUT_DIRECTORY}/functions"
 
 # Build the binary for runner to `mmap`
+"${TRIPLE}gcc" -c \
+  ${CFLAGS} -O3 -static \
```

Comment:
Not sure `-static` makes sense if you're not linking.

Reply:
Yeah, it's a leftover.

Reply:
Drop it?